src/drivers/power_monitor/ina226/ina226.cpp

```diff
@@ -381,47 +244,62 @@ INA226::collect()
 	/* read from the sensor */
 	perf_begin(_sample_perf);
 
-	ret = _bus_volatage = read(INA226_REG_BUSVOLTAGE);
-
-	if (_bus_volatage >= 0) {
-		ret = _power = read(INA226_REG_POWER);
-
-		if (_power >= 0) {
-			ret = _current = read(INA226_REG_CURRENT);
-
-			if (_current >= 0) {
-				ret = _shunt = read(INA226_REG_SHUNTVOLTAGE);
-
-				if (_shunt >= 0) {
-
-					power_monitor_s report{};
-					report.timestamp = hrt_absolute_time();
-					report.voltage_v = (float) _bus_volatage * INA226_VSCALE;
-					report.current_a = (float) _current * _current_lsb;
-					report.power_w   = (float) _power * _power_lsb;
-#if defined(INA226_RAW)
-					report.rconf  = read(INA226_REG_CONFIGURATION);
-					report.rsv    = read(INA226_REG_SHUNTVOLTAGE);
-					report.rbv    = read(INA226_REG_BUSVOLTAGE);
-					report.rp     = read(INA226_REG_POWER);
-					report.rc     = read(INA226_REG_CURRENT);
-					report.rcal   = read(INA226_REG_CALIBRATION);
-					report.me     = read(INA226_REG_MASKENABLE);
-					report.al     = read(INA226_REG_ALERTLIMIT);
-#endif
-
-					/* publish it */
-					_power_monitor_topic.publish(report);
-
-					ret = OK;
-					perf_end(_sample_perf);
-					return ret;
-				}
-			}
-		}
+	_bus_volatage = read(INA226_REG_BUSVOLTAGE);
```

Comment:
```suggestion
	_bus_voltage = read(INA226_REG_BUSVOLTAGE);
```

Reply:
Whoops, I didn't even notice that since it was leftover from the original version of this driver!